### ISSUE TITLE
Jekyll/Liquid: comment content is not completly ignored

### ISSUE THREAD

**Issue by rugk** [jshint/jshint#2970]
Altghough my example in https://github.com/jshint/jshint/issues/2969 worked fine, this causes error messages:

```js
/* jshint ignore:start */

// this is not ignored!!
{% assign somevar = somearray[1] %}

/* jshint ignore:end */
```

**Comment by nicolo-ribaudo** [comment]
I can't reproduce this issue on http://jshint.com
Which version of JSHint are you using? This should have been fixed by gh-2427 (JSHint 2.9.0)

**Comment by rugk** [comment]
Yes, my fault. It does indeed work...
